docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -15,33 +15,39 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.10.0
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
+
+Virtualization:
+
 - [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
 - [VirtualBox](https://www.virtualbox.org/) - Linux or Windows
-- Hyper-V - Windows
+- [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) - Windows
+
+> **NOTE:** To work with Kyma, use only the provided installation and deinstallation scripts. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
 
 ## Set up certificates
 
 Kyma comes with a local wildcard self-signed `server.crt` certificate that you can find under the `/installation/certs/workspace/raw/` directory of the `kyma` repository. Trust it on the OS level for convenience. Alternatively, accept exceptions for each subdomain in your browser as you use Kyma.
```

Comment:
Please remove "Alternatively, accept exceptions for each subdomain in your browser as you use Kyma." or explain that you need to hit every host added to etc/hosts (more than 10)